surface/include/pcl/surface/qhull.h

```diff
@@ -49,14 +49,14 @@
 extern "C"
 {
 #ifdef HAVE_QHULL_2011
```

Comment:
It would be great if you could remove all the checks related to `HAVE_QHULL_2011`, we can assume that the version is much newer than 2011.

Reply:
fixed